cloud/kubernetes/README.md

```diff
@@ -186,11 +221,16 @@ If you created a secure cockroach cluster, run:
 kubectl create -f example_app_secure.yaml
 ```
 
-For every pod being created, you will need to approve its client certificate request:
+When the first pod is being initialized, you will need to approve its client certificate request:
 ```shell
-kubectl certificate approve client.root-example-secure-etc..
+kubectl certificate approve client.root
 ```
 
+If more pods are then added through `kubectl scale deployment example-secure --replicas=X`, the generated
+certificate will be reused.
+**WARNING**: the example app in secure mode should be started with only one replica, or concurrent and
```

Comment:
Oh, sorry, I stopped reading after seeing the `**WARNING**` line and thought you were pointing to the non-working cockroach client invocation in secure mode. Bleh!
I could fix this by either
* using slightly different names in the CSR (but that's annoying if you start a bunch of client replicas, you would need to do something with each CSR)
* having a marker saying the certificate request is "in process" so other replicas could just watch for the secret (I would need something with atomic guarantees in k8s. eg: "create secret if not exists")
* some other type of sharing?